File: ZXYZdatabaseBack/zxyz-project-service/src/main/java/uno/acloud/project/common/ProjectStatus.java
package uno.acloud.project.common;

/**
 * 项目状态。
 *
 * <p>取值口径来自 Flyway 迁移 {@code zxyz-project-service/.../V1__init_schema.sql}：
 * {@code status TINYINT NOT NULL DEFAULT 0 COMMENT '0-正常，1-归档，2-禁用'}。</p>
 */
public final class ProjectStatus {
    /** 正常。 */
    public static final int NORMAL = 0;
    /** 已归档。 */
    public static final int ARCHIVED = 1;
    /** 已禁用。 */
    public static final int DISABLED = 2;

    private ProjectStatus() {
    }
}

File: ZXYZdatabaseFront/src/constants/projectStatus.js
// 项目状态码常量。取值与后端 project.status 一致（Flyway 列注释：0-正常 / 1-归档 / 2-禁用）。
// 本文件是项目状态码的唯一权威来源。
export const PROJECT_STATUS = Object.freeze({
  NORMAL: 0, // 正常
  ARCHIVED: 1, // 已归档
  DISABLED: 2, // 已禁用
})
